docs/install.rst

```diff
@@ -27,26 +44,35 @@ by executing the following command in your virtualenv:
         pip install radical.entk
 
 
-To install a specific branch of EnTK, you will need to clone the repository
-and checkout the branch. You can do so using the following commands:
+To install a specific branch of EnTK, e.g., `devel` instead of using pip
+installation, you will need to clone the repository and checkout the branch.
+You can do so using the following commands:
 
 .. code-block:: bash
 
         git clone https://github.com/radical-cybertools/radical.entk.git
         cd radical.entk
-        git checkout <branch-name>
+        git checkout devel
```

Comment:
I would leave <branch-name> so to remark the idea that this is a general purpose way to checkout whatever branch the user needs. This (used to?) happen when a fix was made specifically for a user before release.

Reply:
Make sense. I will revert it back.